scipy/stats/_continuous_distns.py

```diff
@@ -9309,6 +9309,188 @@ def _updated_ctor_param(self):
         return dct
 
 
+class studentized_range_gen(rv_continuous):
+    r"""A studentized range continuous random variable.
+
+    %(before_notes)s
+
+    See Also
+    --------
+    t: Student's t distribution
+
+    Notes
+    -----
+    The probability density function for `studentized_range` is:
+
+    .. math::
+
+         f(q; k, \nu) = \frac{k(k-1)\nu^{\nu/2}}{\Gamma(\nu/2)
+                        2^{\nu/2-1}} \int_{0}^{\infty} \int_{-\infty}^{\infty}
+                        s^{\nu} e^{-\nu s^2/2} \phi(z) \phi(sq + z)
+                        [\Phi(sq + z) - \Phi(z)]^{k-2} \,dz \,ds
+
+    When :math:`\nu` exceeds 100,000, an asymptotic approximation (infinte
```

Comment:
```suggestion
    When :math:`\nu` exceeds 100,000, an asymptotic approximation (infinite
```
Ugh! Somehow this slipped out of the commit that I specifically intended I changed it in. Will be coming with next round of commits as to not waste CI.